R/derive_vars_dtm.R

```diff
@@ -200,12 +208,20 @@ derive_vars_dtm <- function(dataset, # nolint: cyclocomp_linter
     tmf <- paste0(new_vars_prefix, "TMF")
     warn_if_vars_exist(dataset, tmf)
 
-    dataset <- dataset %>%
-      mutate(!!sym(tmf) := compute_tmf(
-        dtc = !!dtc,
-        dtm = !!sym(dtm),
-        ignore_seconds_flag = ignore_seconds_flag
-      ))
+    tryCatch(
+      dataset <- dataset %>%
+        mutate(!!sym(tmf) := compute_tmf(
+          dtc = !!dtc,
+          dtm = !!sym(dtm),
+          ignore_seconds_flag = ignore_seconds_flag
+        )),
+      "dplyr:::mutate_error" = function(cnd) {
+        cli_abort(
+          message = cnd$parent$message,
+          call = parent.frame(n = 4)
+        )
+      }
+    )
```

Comment:
@zdz2101 was this what you were thinking?  I like this @bundfussr - wondering if we should look for other issues like this in admiral - I'll make an issue.

Reply:
I actually made it into a discussion - https://github.com/pharmaverse/admiral/discussions/2421?converting=1